src/ray/core_worker/fiber.h

```diff
@@ -124,7 +134,7 @@ class FiberState {
               // no fibers can run after this point as we don't yield here.
               // This makes sure this thread won't accidentally
               // access being destructed core worker.
-              fiber_stopped_event_.Notify();
+              fiber_stopped_event->Notify();
```

Comment:
Let's add a `fiber_stopped_event->clear()` to explicitly free the pointer.

Reply:
can't do clear because lambda captures the shared ptr as const. We can mark it mutable but I guess that's overkill.